lib/msal-browser/scripts/README.md

```diff
@@ -0,0 +1,273 @@
+# MSAL Browser Log Decoder
+
+This script helps decode hashed logging strings from the minified MSAL browser bundle back to their original, human-readable messages.
+
+## Background
+
+When MSAL Browser is built in production mode, all logging strings are minified using consistent hashing to reduce bundle size. This means that instead of seeing:
+
+```
+acquireTokenSilent - attempting to acquire token from native platform
+```
+
+Users will see:
+
+```
+ac8ab7
+```
+
+This script allows you to decode these hashes back to the original messages for debugging purposes.
+
+## Features
+
+- **Remote Mapping Fetching**: Automatically fetches log mappings from npm registry for specific versions
+- **Multi-Package Support**: Combines mappings from both `@azure/msal-browser` and `@azure/msal-common` packages
+- **Multiple Input Formats**: Supports various hash input formats (arrays, space-separated, JSON)
+- **Timing Information**: Supports new format with timing data for performance analysis
+- **Verbose Mode**: Optional detailed output for debugging
+- **Caching**: Caches remote mappings locally (24 hours) to reduce network requests
+- **Fallback Support**: Falls back to local mappings when remote fetching fails
+
+## Usage
+
+### Basic Usage
+
+```bash
+# Decode hashes using local mappings
```

Comment:
I'm a little confused by all the different options here and when I should use one over the other. Suggest either adding some explanation for each one if there's value-add for having different formats or deciding on a single standard.